hazelcast/src/main/java/com/hazelcast/map/impl/query/Target.java

```diff
@@ -91,36 +91,7 @@ public void readData(ObjectDataInput in) throws IOException {
         this.mode = TargetMode.valueOf(in.readUTF());
     }
 
-    public static TargetBuilder of() {
-        return new TargetBuilder();
-    }
-
-    public static final class TargetBuilder {
-
-        private TargetMode mode;
-        private Integer partitionId;
-
-        private TargetBuilder() {
-        }
-
-        public TargetBuilder allNodes() {
-            this.mode = TargetMode.ALL_NODES;
-            return this;
-        }
-
-        public TargetBuilder localNode() {
-            this.mode = TargetMode.LOCAL_NODE;
-            return this;
-        }
-
-        public TargetBuilder partitionOwner(int partitionId) {
-            this.mode = PARTITION_OWNER;
-            this.partitionId = partitionId;
-            return this;
-        }
-
-        public Target build() {
-            return new Target(mode, partitionId);
-        }
+    public static Target newTarget(int partitionId) {
```

Comment:
The name should suggest the mode used in the target - e.g. `newPartitionTarget` or `createPartitionTarget`.

Reply:
+1 I'll change it.